backend/integrations/discord/cogs.py

```
@@ -499,17 +531,20 @@ async def _send_onboarding_flow(self, user: discord.abc.User) -> str:
+                if error_msg:
+                    # Handle error (fallback logic)
                     try:
                         await user.send("Couldn't generate a verification link. Please use /verify_github.")
                         await user.send(build_encourage_verification_message(reminder_count=1))
                         await user.send(embed=build_final_handoff_embed())
                     except discord.Forbidden:
-                        logger.warning(f"Cannot DM user {user.id} after auth URL failure (DMs disabled)")
-                    except Exception as e:
-                        logger.exception(f"Failed to send auth failure fallback DM to user {user.id}: {e}")
+                        logger.warning(f"Cannot DM user {user.id} after auth failure (DMs disabled)")
+                    except Exception as send_error:
```

Ruff (0.14.14)

[error] Local variable `send_error` is assigned to but never used

Remove assignment to unused variable `send_error`

(F841)